Tests/UnitTests/Networking/Backend/__Snapshots__/BackendPostReceiptDataTests/iOS15-testPostsSK2XcodeReceiptWithProductDataCorrectly.1.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Awesome!

FYI looks like this is missing iOS 12 through 14 snapshots. Unfortunately that's not caught by CI because when they're missing, they get generated (making the test fail), but then the test is retried and it passes as a false positive.

_I have a [PR to fix that](https://github.com/pointfreeco/swift-snapshot-testing/pull/768)_

Reply:
Added